cmd/storagenode/main.go

```diff
@@ -292,6 +304,58 @@ func cmdDiag(cmd *cobra.Command, args []string) (err error) {
 	return err
 }
 
+func dashCmd(cmd *cobra.Command, args []string) (err error) {
+	// create new client
+	ctx := context.Background()
+	lc, err := psclient.NewLiteClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	stream, err := lc.Dashboard(ctx)
+	if err != nil {
+		return err
+	}
+
+	for {
+		data, err := stream.Recv()
+		if err == io.EOF {
+			break
+		}
+
+		if err != nil {
+			return err
+		}
+
+		clr()
+		color.Green("\n\033[1mStorage Node Dashboard Stats\033[1m\n")
```

Comment:
Oh, this will definitely break on Windows. Use the appropriate stuff from `color` as necessary rather than directly using escape codes.